contributors/THIRD-PARTY-EXTENSION.md

```diff
@@ -0,0 +1,104 @@
+## Introduction
+
+Welcome to the guide for developing external extensions in the `create-eth` repository. This document outlines the workflow for creating and testing your extensions.
+
+## Workflow Overview
+
+### Phase 1: Initial Setup and Development
+
+1. **Clone the `create-eth` Repository:**
+
+   ```bash
+   git clone https://github.com/scaffold-eth/create-eth.git
+   cd create-eth
+   ```
+
+2. **Run the Build Script:**
+
+   ```bash
+   yarn build:dev
+   ```
+
+3. **Run the CLI to Create a New Instance:**
+
+   ```bash
+   yarn cli
+   ```
+
+   This command will create a **new instance** which is similar to running `npx create-eth@latest`
+
+   The name mentioned for "Your project name" question will be used as the **extension name**. For example, if you provide `eip`, the final extension name will be `eip`.
+
+4. **Develop the Extension:**
+   Add new files within the new instance directory. For example:
+
+   ```bash
+   # In new terminal window
+   cd eip
+   mkdir -p packages/nextjs/app/my-eip
+   touch packages/nextjs/app/my-eip/page.tsx
+   # add content to my-eip/page.tsx
+   ```
+
+   > **NOTE**: Only adding of new files / directories is allowed while creating extension. If you try to overwrite existing files\*, it wont be reflected instead `yarn create-extension {projectName}` should guide you with info for the respective file. Checkout **Special Files** point for more info.
+
+5. **Commit Changes to the Instance Repository:**
+
+   ```bash
+   git add .
+   git commit -m "Add my-page extension"
+   ```
+
+6. **Create the Extension:**
```

Comment:
Do we need to mention that user need to run `create-extension` from `create-eth` repo?

Reply:
[c7b5b43](https://github.com/scaffold-eth/create-eth/pull/84/commits/c7b5b436178fbb1cd01b2bb9a52623eebb038dcd)